src/app_service/service/community_tokens/service.nim

```diff
@@ -139,3 +147,12 @@ QtObject:
       if token.symbol == symbol:
         return token
 
+  proc getFiatValue*(self: Service, cryptoBalance: string, cryptoSymbol: string): string =
+    if (cryptoBalance == "" or cryptoSymbol == ""):
+      return "0.00"
+
+    let currentCurrency = self.settingsService.getCurrency()
+
+    let price = self.tokenService.getTokenPrice(cryptoSymbol, currentCurrency)
+    let value = parseFloat(cryptoBalance) * price
+    return fmt"{value:.2f}"
```

Comment:
How is this supposed to be correctly formatted wrt to decimal separator when we are returning a string here?

Reply:
This should return a `double` and formatted properly in QML

Reply:
There is a ticket for this for next milestone: https://github.com/status-im/status-desktop/issues/10035

Reply:
Cool, thanks :)